frontend/llvm/src/import/bundle.cpp

```diff
@@ -335,8 +354,13 @@ ar::Function* BundleImporter::translate_library_function(llvm::Function* fun) {
       buf << "llvm function " << fun->getName().str() << " and ar intrinsic "
           << ar_fun->name() << " have a different type";
     }
-
-    throw ImportError(buf.str());
+    // We no longer throw but give warning.
+    //   throw ImportError(buf.str());
+    std::cerr << "Warning:" << buf.str() << "\n";
```

Comment:
I would just remove that comment. Also nitpicking: add a space after `Warning:`

Reply:
Will do.